### PR TITLE
Hiding Semantics Practice Page: Add missing word "that"

### PR DESCRIPTION
**Pull request by frozenzia**
___
[WAI Preview Link](https://deploy-preview-222--aria-practices.netlify.app/ARIA/apg) _(Last built on Fri, 09 Jun 2023 20:40:42 GMT)._

### CODE REVIEW

**Comment by howard-e** [comment]
This change (https://deploy-preview-222--aria-practices.netlify.app/aria/apg/practices/hiding-semantics/#presentation_role_effects) seems good to me.

cc @mcking65 @a11ydoer

File: content/practices/hiding-semantics/hiding-semantics-practice.html [hunk 1]
```diff
@@ -63,7 +63,7 @@ <h2>Effects of Role <code>presentation</code></h2>
           <li>The element’s implied ARIA role and any ARIA states and properties associated with that role are hidden from assistive technologies.</li>
           <li>
             Text contained by the element, i.e., inner text, as well as inner text of all its descendant elements remains visible to assistive technologies.
-            Of course, text is explicitly hidden, e.g., styled with <code>display: none</code> or has <code>aria-hidden="true"</code> is not visible to assistive technologies.
+            Of course, text which is explicitly hidden, e.g., styled with <code>display: none</code> or has <code>aria-hidden="true"</code> is not visible to assistive technologies.
```
Comment (mcking65) [inline content/practices/hiding-semantics/hiding-semantics-practice.html:66]:
```suggestion
            Of course, text that is explicitly hidden, e.g., styled with <code>display: none</code> or has <code>aria-hidden="true"</code>, is not visible to assistive technologies.
```

**Review by mcking65 — CHANGES_REQUESTED** [review]
@frozenzia, great catch, thank you!!

I used the suggestion feature of the PR to request two changes.

* change from "which" to "that" because the clause is essential.
* Add a comma after the independent "for example" clause.

File: content/practices/hiding-semantics/hiding-semantics-practice.html (hunk 1 above)
Comment (frozenzia) [reply content/practices/hiding-semantics/hiding-semantics-practice.html:66]: Alright, your grammar knowledge is better than mine - I never bothered to understand the which/that difference.

**Comment by frozenzia** [comment]
@ccanash , I would if I could, but I see

"Merging is blocked -- The base branch restricts merging to authorized users."

Sorry if I'm just being dense, somehow. I'm not very experienced at all in contributing to projects like this, and in the past when I *have*, the merging has always been handled by the "team in charge". But I'm definitely willing to help if we can just figure out what I should do at this point.

And now I've written all that out it occurs to me that you asked me to "apply the change" that Matt suggested, not do any sort of merging. I'm sorry, but I'm still not seeing where/what I should do.

**Comment by frozenzia** [comment]
@ccanash, had another stab at this after my workday, think I understand now what it was I was supposed to do. "Unresolved" Matt's comment, and that let me then commit the change he suggested. Hope that's all good now.

**Review by mcking65 — APPROVED** [review]

**Comment by mcking65** [comment]
Thank you @frozenzia for finding this problem and submitting a fix!!

**Pull request merged**